x-pack/plugins/telemetry_collection_xpack/schema/xpack_root.json

```diff
@@ -0,0 +1,2691 @@
+        },
+        "expiry_date_in_millis": {
+          "type": "long"
+        },
+        "issue_date_in_millis": {
+          "type": "long"
+        },
+        "start_date_in_millis": {
+          "type": "long"
+        },
+        "issued_to": {
+          "type": "keyword"
+        },
+        "issuer": {
+          "type": "keyword"
+        },
+        "status": {
+          "type": "keyword"
+        },
+        "type": {
+          "type": "keyword"
+        },
+        "max_nodes": {
+          "type": "long"
+        },
+        "max_resource_units": {
+          "type": "long"
+        }
+      }
+    },
+    "stack_stats": {
+      "properties": {
+        "xpack": {
+          "properties": {
+            "aggregate_metric": {
+              "properties": {
+                "enabled": {
+                  "type": "boolean"
+                },
+                "available": {
+                  "type": "boolean"
+                }
+              }
+            },
+            "runtime_fields": {
+              "properties": {
+                "enabled": {
+                  "type": "boolean"
+                },
+                "available": {
+                  "type": "boolean"
+                },
+                "field_types": {
+                  "type": "array",
+                  "items": {
+                    "properties": {
+                      "name": {
+                        "type": "keyword"
+                      },
+                      "count": {
+                        "type": "long"
+                      },
+                      "index_count": {
+                        "type": "long"
+                      },
+                      "scriptless_count": {
+                        "type": "long"
+                      },
+                      "shadowed_count": {
+                        "type": "long"
+                      },
+                      "lang": {
+                        "type": "array",
+                        "items": {
+                          "type": "keyword"
+                        }
+                      },
+                      "lines_max": {
+                        "type": "long"
+                      },
+                      "lines_total": {
+                        "type": "long"
+                      },
+                      "chars_max": {
+                        "type": "long"
+                      },
+                      "chars_total": {
+                        "type": "long"
+                      },
+                      "source_max": {
+                        "type": "long"
+                      },
+                      "source_total": {
+                        "type": "long"
+                      },
+                      "doc_max": {
+                        "type": "long"
+                      },
+                      "doc_total": {
+                        "type": "long"
+                      }
+                    }
+                  }
```

Comment:
I got these ones from the Java code. I couldn't create a use-case where we got these.